gems/pending/appliance_console/internal_database_configuration.rb

```diff
@@ -163,7 +163,7 @@ def update_fstab
 
     def prep_data_directory
       # initdb will fail if the database directory is not empty or not owned by the PostgresAdmin.user
-      # May need to create the data dir here?
+      FileUtils.mkdir(PostgresAdmin.data_directory) unless Dir.exist?(PostgresAdmin.data_directory)
```

Comment:
Every single thing in this method references PostgresAdmin class that I feel like it should *live* in the PostgresAdmin class.